warn: interrupts.int_master is deprecated. `int_master` is now called `int_requestor`
warn: interrupts.int_slave is deprecated. `int_slave` is now called `int_responder`
warn: No dot file generated. Please install pydot to generate the dot file and pdf.
warn: DRAM device capacity (8192 Mbytes) does not match the address range assigned (2048 Mbytes)
panic: ListenSocket(listen): bind() failed!
Memory Usage: 2196192 KBytes
Program aborted at tick 0
--- BEGIN LIBC BACKTRACE ---
build/X86/gem5.opt(+0x364490)[0x5585885af490]
build/X86/gem5.opt(+0x3789ce)[0x5585885c39ce]
/lib/x86_64-linux-gnu/libpthread.so.0(+0x143c0)[0x7f1b633383c0]
/lib/x86_64-linux-gnu/libc.so.6(gsignal+0xcb)[0x7f1b628d903b]
/lib/x86_64-linux-gnu/libc.so.6(abort+0x12b)[0x7f1b628b8859]
build/X86/gem5.opt(+0x327fe5)[0x558588572fe5]
build/X86/gem5.opt(+0x15a0dac)[0x5585897ebdac]
build/X86/gem5.opt(+0x15a02b6)[0x5585897eb2b6]
build/X86/gem5.opt(+0x39d94d)[0x5585885e894d]
build/X86/gem5.opt(+0x39daf7)[0x5585885e8af7]
build/X86/gem5.opt(+0x4c6f80)[0x558588711f80]
build/X86/gem5.opt(+0x4c72b8)[0x5585887122b8]
build/X86/gem5.opt(+0x54d584)[0x558588798584]
build/X86/gem5.opt(+0xceacfa)[0x558588f35cfa]
build/X86/gem5.opt(+0x3ea78f)[0x55858863578f]
/lib/x86_64-linux-gnu/libpython3.8.so.1.0(+0x2a8748)[0x7f1b635ef748]
/lib/x86_64-linux-gnu/libpython3.8.so.1.0(_PyObject_MakeTpCall+0xab)[0x7f1b635efb2b]
/lib/x86_64-linux-gnu/libpython3.8.so.1.0(+0x2a8df0)[0x7f1b635efdf0]
/lib/x86_64-linux-gnu/libpython3.8.so.1.0(+0x74d6d)[0x7f1b633bbd6d]
/lib/x86_64-linux-gnu/libpython3.8.so.1.0(_PyEval_EvalFrameDefault+0x7d86)[0x7f1b633c3ef6]
/lib/x86_64-linux-gnu/libpython3.8.so.1.0(+0x8006b)[0x7f1b633c706b]
/lib/x86_64-linux-gnu/libpython3.8.so.1.0(+0x74d6d)[0x7f1b633bbd6d]
/lib/x86_64-linux-gnu/libpython3.8.so.1.0(_PyEval_EvalFrameDefault+0x7d86)[0x7f1b633c3ef6]
/lib/x86_64-linux-gnu/libpython3.8.so.1.0(_PyEval_EvalCodeWithName+0x8fb)[0x7f1b63511e4b]
/lib/x86_64-linux-gnu/libpython3.8.so.1.0(PyEval_EvalCodeEx+0x42)[0x7f1b635121d2]
/lib/x86_64-linux-gnu/libpython3.8.so.1.0(PyEval_EvalCode+0x1f)[0x7f1b635125bf]
/lib/x86_64-linux-gnu/libpython3.8.so.1.0(+0x1cfc01)[0x7f1b63516c01]
/lib/x86_64-linux-gnu/libpython3.8.so.1.0(+0x25f547)[0x7f1b635a6547]
/lib/x86_64-linux-gnu/libpython3.8.so.1.0(+0x74d6d)[0x7f1b633bbd6d]
/lib/x86_64-linux-gnu/libpython3.8.so.1.0(_PyEval_EvalFrameDefault+0x12fd)[0x7f1b633bd46d]
/lib/x86_64-linux-gnu/libpython3.8.so.1.0(_PyEval_EvalCodeWithName+0x8fb)[0x7f1b63511e4b]
/lib/x86_64-linux-gnu/libpython3.8.so.1.0(_PyFunction_Vectorcall+0x94)[0x7f1b635ef124]
--- END LIBC BACKTRACE ---
